pkgs/applications/networking/browsers/ephemeral/default.nix

```diff
@@ -20,13 +20,13 @@
 
 stdenv.mkDerivation rec {
   pname = "ephemeral";
-  version = "6.4.1";
+  version = "7.0.4";
```

Comment:
seems like this unrelated commit snuck in?

Reply:
Thanks. Just removed it.